features/step_definitions/meta_steps.rb

```diff
@@ -114,3 +114,21 @@
   end
 end
 
+Given /^I run the steps #{NUMBER} times or exit when co machine-api is degraded:$/ do |num, steps_string|
```

Comment:
This is very specific to machine-api, better to move it to the machine_api.rb or elsewhere than meta_steps.rb